packages/repack/ios/ScriptConfig.h

```diff
@@ -20,6 +20,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (readonly) NSNumber *timeout;
 @property (readonly) NSString *verifyScriptSignature;
 @property (readonly) NSString *uniqueId;
+@property (readonly) NSString *sourceUrl;
```

Comment:
nit: a further improvement, you can probably make all of those properties nonatomic, https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ObjectiveC/Chapters/ocProperties.html

Reply:
what would be the benefit of that?

Reply:
_speed_ 🚅, if you add nonatomic the access should be faster

Reply:
will do in a separate PR to keep this on topic, thx!